src/main/java/edu/asu/plp/tool/backend/isa/events/RegWatchRequestEvent.java

```diff
@@ -1,13 +1,16 @@
 package edu.asu.plp.tool.backend.isa.events;
 
+// Class to watch register values
 public class RegWatchRequestEvent {
 
 	private String registerName;
 
+	// Constructor with registerName as a parameter
```

Comment:
JavaDoc format
```
/**
 * @brief Function purpose
 * @param param1 meanning of param1
 * @param param2 menning of param2
 * @return 1 on success, otherwise false. (if this funcion has a return value)
 */
```